taxi/models.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    class Mete:

```

Comment:
There is a typo in the class name `Mete`. It should be `Meta` to correctly define the model's metadata.